verkle-trie/src/proof/stateless_updater.rs

```diff
@@ -35,14 +36,21 @@ pub(crate) fn update_root<C: Committer>(
     updated_values: Vec<Option<[u8; 32]>>,
     root: Element,
     committer: C,
-) -> Element {
-    assert_eq!(values.len(), updated_values.len());
-    assert_eq!(keys.len(), updated_values.len());
+) -> Result<Element, VerkleError> {
+    if !values.len() == updated_values.len() {
+        return Err(VerkleError::UnexpectedUpdatedLength);
```

Comment:
Can you modify the `UnexpectedUpdatedLength` variant to take also take in two integers; values.len and updated_values.len ?

This means that error messages can become more descriptive in the future